### PR TITLE
Finished array reduce

### PR DESCRIPTION
**Pull request by josh-tjiauw**
image

### CODE REVIEW

File: array-reduce/main.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const balance = account.reduce((accumulator, currentValue) => {
+  if(currentValue.type === 'withdrawal'){
+    return accumulator -= currentValue.amount
+  }
+  return accumulator += currentValue.amount
+}, 0);
+console.log(balance)
+
+const composite = traits.reduce((accumulator, currentValue) => {
+  for (let x in traits) {
```
Comment (Zhiwutian) [inline array-reduce/main.js:35]: you should not be using a for in loop here. this should be done by the looping functionality of reduce.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Comment by josh-tjiauw** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**